index.js

```diff
@@ -14,15 +16,21 @@ var anymatch = function(criteria, value, returnIndex, startIndex, endIndex) {
   }
   startIndex = startIndex || 0;
   var string = value[0];
+  var altString;
+  if (platform === 'win32' && typeof string === 'string') {
```

Comment:
nonetheless I can do something like `var backslashPaths = path.sep === '\\'` at the top which would be more direct and accurate than `isWindows`